server/config/nl_page/chart_titles_by_sv.json

```diff
@@ -1061,7 +1061,7 @@
   "NumberOfMonths_WetBulbTemperature_35COrMore_RCP85_MeanRelativeHumidity": "Number of Months Reaching Wet Bulb Temperature based on RCP 8.5 Mean Relative Humidity",
   "NumberOfMonths_WetBulbTemperature_35COrMore_RCP85_MinRelativeHumidity": "Number of Months Reaching Wet Bulb Temperature based on RCP 8.5 Min Relative Humidity",
   "PalmerDroughtSeverityIndex_Atmosphere": "Palmer Drought Severity Index",
-  "Percent_Person_18OrMoreYears_WithHighBloodPressure_ReceivedTakingBloodPressureMedication": "Percent of Adults with Blood Pressure Who Take Medication",
+  "Percent_Person_18OrMoreYears_WithHighBloodPressure_ReceivedTakingBloodPressureMedication": "Percent of  Population With High Blood Pressure Who Are Taking Blood Pressure Medication  Among Population Who Are 18 Years or Older in Age;",
```

Comment:
Please revert this?

Reply:
sorry this is because I had run the sync script with https://docs.google.com/spreadsheets/d/1lmNAnqECpcvkuOlIkdo50Ve1KAalOoyP_lUlOuLmIAU/edit#gid=772929381 and they had the new titles there. I changed the titles in the spreadsheet to match the old ones too